vendor/bat-native-ads/src/bat/ads/internal/ad_diagnostics/ad_diagnostics_catalog_last_updated.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_DIAGNOSTICS_AD_DIAGNOSTICS_CATALOG_LAST_UPDATED_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_DIAGNOSTICS_AD_DIAGNOSTICS_CATALOG_LAST_UPDATED_H_
+
+#include <string>
+
+#include "bat/ads/internal/ad_diagnostics/ad_diagnostics_entry.h"
+
+namespace ads {
+
+class AdDiagnosticsCatalogLastUpdated : public AdDiagnosticsEntry {
```

Comment:
nit: `CatalogLastUpdatedAdDiagnosticsEntry` instead of `AdDiagnosticsCatalogLastUpdated` to follow on from `AdDiagnosticsEntry`

Reply:
done